### ISSUE TITLE
Family history of hereditary hemorrhagic telangiectasia (HHT)

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6577]
**Preferred term label:**  Family history of hereditary hemorrhagic telangiectasia (HHT)

**Synonyms**
Family history of Osler-Weber-Rendu disease

**Definition (free text, please give PubMed ID)**
A first-degree relative with hereditary hemorrhagic telangiectasia (HHT)

**Parent term (use hpo.jax.org/app)**
Family history HP:0032316

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Hereditary hemorrhagic telangiectasia (HHT) ORPHA:774

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
The Family History section of the HPO is meant to record general inexact histories that are typical of many patient histories. It should not contain terms for exact diagnoses like this, because that would lead to an explosion of terms that would mirror a disease ontology. I would suggested coding this using Mondo (https://www.ebi.ac.uk/ols/ontologies/mondo) in a special field of your database.
